getting_started/first_2d_game/05.the_main_game_scene.rst

```diff
@@ -337,11 +339,11 @@ Note that a new instance must be added to the scene using ``add_child()``.
         mob.position = mob_spawn_location.position
 
         # Add some randomness to the direction.
-        direction += rand_range(-PI / 4, PI / 4)
+        direction += randf_range(-PI / 4, PI / 4)
         mob.rotation = direction
 
         # Choose the velocity.
-        var velocity = Vector2(rand_range(150.0, 250.0), 0.0)
+        var velocity = Vector2(randi_range(150, 250), 0.0)
```

Comment:
@BryanQuigley As Vector2 works with float as well, I believe this should be `randf_range` also.

Reply:
I did test this one with either option, and I really can't tell the difference.

Reply:
Changed it back to be consistent with the original code and other languages.